docs/howto/prepare-and-run.md

```diff
@@ -0,0 +1,70 @@
+(prepare-and-run)=
+# How to prepare and run a SQLair query
+
+A SQLair query is just like a regular SQL query, except that it contains special
+input and output expressions. These indicate to SQLair the Go objects that your
+inputs and outputs correspond to. To create and run a SQLair query there are
+several steps:
+1. Ensure the types you want to map to the query inputs and outputs are in the
+   right form for SQLair.
+2. Write your query with SQLair syntax.
+3. Prepare your query with `sqlair.Prepare`.
+4. Run your query on a database or transaction.
+
+## Prepare your Go types
+
+See {ref}`types` for how to prepare your types.
+
+## Write your query
+
+SQLair queries are regular SQL queries with special input and output expressions
```

Comment:
This repeats content from the intro. FWIW it feels more relevant here, so maybe you can skip it in the intro.

Reply:
Good point